client/app/utils/metaTagsManager.js

```diff
@@ -6,15 +6,10 @@ const MetaTagsManager = {
    * @returns {String} - CSRF Token.
    */
   getCSRFToken() {
-    const metas = document.getElementsByTagName('meta');
-    for (let i = 0; i < metas.length; i++) {
-      const meta = metas[i];
-      if (meta.getAttribute('name') === 'csrf-token') {
-        return meta.getAttribute('content');
-      }
-    }
+    const token = Array.from(document.getElementsByTagName('meta'))
```

Comment:
You're right. We don't have lodash in this project.

```
_.find(document.getElementsByTagName('meta'), tag => tag.name === 'csrf-token')
```
![2015-10-16_11-58-48](https://cloud.githubusercontent.com/assets/1118459/10550715/4f4cb100-73fd-11e5-881a-c04f38857e7b.png)
s project.

I really like Lodash. I wouldn't suggest this otherwise. That being said, we don't *need* it give some features in ES6.

Because I want this example to be a real starter kit showing off Webpack/Npm/React integration, I think this is worth considering.

Let's see what @mapreal19, @alexfedoseev, @samnang, @dylangrafmyre Think about this one.